authorizations/authorization-ranger/src/main/java/org/apache/gravitino/authorization/ranger/RangerHadoopSQLMetadataObject.java

```diff
@@ -35,7 +35,9 @@ public enum Type implements AuthorizationMetadataObject.Type {
     /** A table is mapped the table of relational data sources like Apache Hive, MySQL, etc. */
     TABLE(MetadataObject.Type.TABLE),
     /** A column is a sub-collection of the table that represents a group of same type data. */
-    COLUMN(MetadataObject.Type.COLUMN);
+    COLUMN(MetadataObject.Type.COLUMN),
+    /** A path is mapped the path of storages like HDFS, S3 etc. */
+    PATH(MetadataObject.Type.FILESET);
```

Comment:
Please remove `PATH`. Because `RangerHDFSMetadataObject.type` already has it.

Reply:
removed